### ISSUE TITLE
Discord RPC Support?

### ISSUE THREAD

**Issue by CrankySupertoon** [SourMesen/Mesen#820]
Hi I was curious if their are any plans for Discord RPC Support. 

Seeing I did most of the Project64 RPC code figured I would help here https://github.com/project64/project64/pull/1665

If there is any reason you don’t want me to do so please DM Me. Does Mesen-S use the same hooks for reading the game names because then it could be extremely easy

**Comment by SourMesen** [comment]
No, I don't really plan on adding support for it.  I originally did the work to add support for it myself, but decided against adding it in the end.  One of the C# implementations was reasonably small, but did not work in Mono on Linux . The C++ SDK on the other hand is just too large to justify for such a small feature - 17k lines of code to compile, maintain, potentially update as discord changes their API, to essentially display a line of text under a person's name in discord :p

I understand some people might want a feature such as this - but as the codebase (and options) grow over the years, I've been trying more and more to avoid feature creep as much as I can, especially since I'm the person who gets stuck maintaining all of these features in the end :p  At this point in time, I would rather not add support for this.

**Comment by CrankySupertoon** [comment]
ill probally take a stab at it but yeah the issue probally will be what you said

**Comment by CrankySupertoon** [comment]
it will be in a PR if im successful
